tests/wpunit/PostObjectQueriesTest.php

```diff
@@ -1174,7 +1174,8 @@ public function testPostByQueryWithIDForADifferentType() {
 		/**
 		 * This should return an error as we tried to query for a post using a Page ID
```

Comment:
Should update this comment since it should now _not_ throw an error.

Reply:
Good call. Thanks!

Reply:
@CodeProKid @jasonbahl Why is bad to throw an error here? If you just return `null` the user won't know what went wrong.